plugins/api-docs/package.json

```diff
@@ -30,7 +30,7 @@
     "clean": "backstage-cli clean"
   },
   "dependencies": {
-    "@asyncapi/react-component": "^0.23.0",
+    "@asyncapi/react-component": "^1.0.0-next.21",
```

Comment:
Hey @Rugvip thanks for your feedback 🙂 According to [this](https://github.com/asyncapi/asyncapi-react/pull/493#issuecomment-986639603) comment I would assume that the release for 1.0 should happen soon. We just merged [this](https://github.com/asyncapi/asyncapi-react/pull/493) fix and I updated this PR to include it so now there are no global styles affecting other components, as everything is scoped within AsyncAPI. From my perspective it would be ok to go with this pre-release version. I would prefer to do another version bump instead of having this PR on hold 🙂.

Reply:
Alright, thank you! 👍 

If you'd prefer that way I'd say we're good to :shipit: 😁

Reply:
awesome 🙂 As I don't have write access could you merge the PR @Rugvip or is there anything I can do from my side ? 🙂